cypress/page/shopping-cart.page.ts

```diff
@@ -3,7 +3,7 @@ class ShoppingCartPage {
     private processToCheckoutButton: string;
 
     constructor() {
-        this.processToCheckoutButton = ".cart_navigation > .button > span";
+        this.processToCheckoutButton = '.cart_navigation > a[title="Proceed to checkout"] > span';
```

Comment:
what about ` a[title="Proceed to checkout"]` or `.cart_navigation a[title="Proceed to checkout"]`

Reply:
Done!